README.rst

```diff
@@ -64,14 +64,14 @@ Used to enable special encoding of "unsafe" HTML characters into safer Unicode s
 
 ensure_ascii
 -------------
-Limits output to ASCII and escapes all extended characters above 127. Default is true. If your end format supports UTF-8 setting this option to false is highly recommended to save space:
+Limits output to ASCII and escapes all extended characters above 127. Default is ``True``. If your end format supports UTF-8, setting this option to false is highly recommended to save space:
 
 .. code-block:: python
 
-    >>> ujson.dumps(u"\xe5\xe4\xf6")
+    >>> ujson.dumps("åäö")
```

Comment:
Hello everyone! 

I have a **problem** installing / building ujson in a ubuntu18.04 Docker container for arm64/aarch64. 

This has started with this change, adding the special characters. 
It seems that in setup.py the reading of README.rst has problems with the encoding. 

The **error** I get is 
```
Traceback (most recent call last):
  File "setup.py", line 38, in <module>
    long_description = f.read()
  File "/usr/lib/python3.6/encodings/ascii.py", line 26, in decode
    return codecs.ascii_decode(input, self.errors)[0]
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 2116: ordinal not in range(128)
``` 

A possible **solution** is (not sure what is the best way) to change [Line 37 in setup.py](https://github.com/ultrajson/ultrajson/blob/62b89200c69723bfe1ae3ff56c94a70c99459a9d/setup.py#L37) to `with open("README.rst", "rt", encoding="utf-8") as f:`

To **reproduce** this on a x86 machine (with docker and qemu installed) build the following Dockerfile and start the container: 

```
FROM ubuntu:18.04@sha256:c57d3145ffd3adb9e4dea2704c08d4d027dd57cb6e30d69dce6c1d14bd38fe5c as ujsontest

RUN apt-get update
RUN apt-get -y upgrade
RUN apt -y install git vim build-essential python3 python3-dev python3-pip

RUN git clone https://github.com/ultrajson/ultrajson.git
RUN pip3 install wheel
WORKDIR /ultrajson

RUN pip3 install --upgrade setuptools

ENTRYPOINT python3 setup.py bdist_wheel
```

Run: 
```
docker build -t ujsonencodingerror -f Dockerfile .
docker run --rm ujsonencodingerror
```

**Other setups build / install without problems**. (i.e. Ubuntu 18.04 installed on Raspberry Pi 4 (aarch64) and Debian in a x86 VM). 

I would appreciate if someone could look into this. If additional information is necessary, please contact me. 
Thank! 
Cheers,
David